xds/data/orca/v3/orca_load_report.proto

```diff
@@ -14,8 +14,9 @@ import "validate/validate.proto";
 
 message OrcaLoadReport {
   // CPU utilization expressed as a fraction of available CPU resources. This
-  // should be derived from the latest sample or measurement.
-  double cpu_utilization = 1 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
+  // should be derived from the latest sample or measurement. The value may be
+  // larger than 1.0 when the usage exceeds the soft limit.
```

Comment:
"exceeds a reporter dependent notion of soft limits". I think it's legit, just want to be clear who is specifying the soft limit.

Reply:
Updated.